packages/cxl-ui/scss/global/cxl-card.scss

```diff
@@ -0,0 +1,69 @@
+cxl-card {
+  box-sizing: border-box;
+  padding: var(--lumo-space-s) var(--lumo-space-m);
+  margin: var(--lumo-space-m);
+  overflow: hidden;
+  font-family: inherit;
+  text-overflow: ellipsis;
+  background: var(--lumo-base-color);
+  border: 1px solid var(--lumo-contrast-10pct);
+  border-radius: var(--lumo-border-radius);
+
+  &:hover {
+    border-color: var(--lumo-primary-color);
+  }
+
+  iron-icon {
+    --iron-icon-height: var(--lumo-icon-size-s);
+    --iron-icon-width: var(--iron-icon-height);
+  }
+
+  .entry-header {
+    margin: 0;
+    font-family: inherit;
+    font-style: normal;
+
+    .headshot {
+      width: 120px;
+      height: 120px;
+      margin: 0 auto;
+      border: 5px solid var(--lumo-base-color);
+      border-radius: 100%;
+    }
+
+    .company-logo {
```

Comment:
Generic `cxl-card` wouldn't know anything about company-logo, this would be some specific theme only.

Also, let's initially assume just one image, `.thumbnail`.

Reply:
Moved to theme=cxl-testimonial